docs/style.md

```diff
@@ -0,0 +1,28 @@
+# Style Guide
+
+## Generic
+
+* This guide can be ignored for data files that are not to be viewed in an editor.
+* 120 character per line for all source files.
+* Avoid white space errors like trailing spaces.
+
+
+## Python Code
+
+* We mostly follow [PEP8](https://peps.python.org/pep-0008/). Read below to see the differences.
+* 120 characters per line. PyCharm does this automatically, other editors can be configured for it.
```

Comment:
i hate limiting line lengths, its more importent to create good readable code. even if some parameters are not directly visable on screen. it can still be better readable then splitng a function call over multiple lines

Also 120 is a really old value people used when they resolutuions of max 1024px

Reply:
120 will make two code windows fit side-by-side on modern screens. 80 was the old one.

Reply:
I use a 120 character limit for all my projects, and it's the default in JetBrains products, which we recommend.